connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignor.java

```diff
@@ -753,6 +617,146 @@ private ConnectorsAndTasks assignment(Map<String, ConnectorsAndTasks> memberAssi
         ).build();
     }
 
+    /**
+     * Revoke connectors and tasks from each worker in the cluster until no worker is running more than it would be if:
+     * <ul>
+     *     <li>The allocation of connectors and tasks across the cluster were as balanced as possible (i.e., the difference in allocation size between any two workers is at most one)</li>
+     *     <li>Any workers that left the group within the scheduled rebalance delay permanently left the group</li>
```

Comment:
TBH, the javadoc can be more clear if we remove all these 3 bullets, because I felt more confused after reading these 3 bullets. So, maybe change to:
`Revoke connectors and tasks from each worker in the cluster until no worker is running more than it would be`

WDYT?

Reply:
Fine with simplifying this 👍 
I updated it to `Revoke connectors and tasks from each worker in the cluster until no worker is running more than it would be with a perfectly-balanced assignment.`; LMKWYT

Reply:
LGTM